src/main/java/Main.java

```diff
@@ -0,0 +1,117 @@
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.apache.commons.io.FileUtils.readLines;
+
+public class Main {
+
+
+    public static void main(String[] args) {
+
+        String path;
+
+        if (args.length==0 || args[0].isEmpty()) {
+            Scanner input = new Scanner(System.in);
+            System.out.println("Enter input file name with full path to it : ");
+            path = input.nextLine();
+        }else {
+
+            path = args[0];
+        }
+        System.out.println(path);
+
+        ArrayList<String> rows = new ArrayList<>();
+        ArrayList<Integer> digits = new ArrayList<>();
+
+        //  TODO: use FileUtils from Apache Commons library (add with maven) to replace below loop
+        //  https://stackoverflow.com/a/4580340
+        try {
+            File inputFile = new File(path);
+//          *********************************************************************************
+//          CLARIFICATION NEEDED: I used  FileUtils.readLines and got List<String> lines
```

Comment:
No one told that you have to divide list, just use it as it is. Pay attention, that for all three methods I asked you to use List<**String**> as input parameter.

Reply:
Done